### PR TITLE
Value update cb error + UI

### PR DESCRIPTION
**Pull request by daniel0226**

### CODE REVIEW

File: app/components/FormLine.js [hunk 1]
```diff
@@ -23,9 +20,33 @@ const FormLine = ({label, value, valueUpdateCB}) => {
...
+            width:              '70%',
+            marginBottom:       10,
+            paddingTop:         6,
+            paddingBottom:      6,
+            paddingLeft:        '3.333%',
+            paddingRight:       '3.333%',
+        },
+
+        textContainer:
+        {
```
Comment (Rayjk123) [inline app/components/FormLine.js]: styling. Would fix this later with prettier or tslint

Comment (daniel0226) [reply app/components/FormLine.js]: It's just the basic style for now.

File: app/components/FormLine.js [hunk 2]
```diff
@@ -23,9 +20,33 @@ const FormLine = ({label, value, valueUpdateCB}) => {
...
+            width:              '70%',
+            overflow:           'scroll',
+        },
+    });
+
     return (
-        <View style={{flexDirection:"row"}}>
-            <Text style={{paddingRight: 30}}>
+        <View style={inputStyle.inputContainer}>
+            <Text style={inputStyle.textContainer}>
```
Comment (Rayjk123) [inline app/components/FormLine.js]: Nice~ I likey

File: app/components/NumberTextInput.js [hunk 3]
```diff
@@ -1,28 +1,45 @@
...
+        if (!value)return;
 
         const regexNotNumberOrDot = /[^0-9.]/g;
         const regexLeadingNumber = /^0+/;
 
         value.replace(regexNotNumberOrDot, '');
         value.replace(regexLeadingNumber, '');
 
-        return valueUpdateCB(value);
+        return valueUpdateCB(label, value);
```
Comment (Rayjk123) [inline app/components/NumberTextInput.js]: Oh was this the issue?

File: app/reducers/index.js [hunk 4]
```diff
@@ -18,7 +18,7 @@ const initialState = {
 };
 
 export default function mainReducer(state = initialState, action) {
-    switch (action) {
+    switch (action.type) {
         case constants.FORM_UPDATE:
```
Comment (Rayjk123) [inline app/reducers/index.js]: Yep good call

**Pull request merged**